sretoolbox/container/image.py

```diff
@@ -8,7 +8,7 @@
 
 import requests
 
-from requests.exceptions import HTTPError
+from requests.exceptions import HTTPError, ConnectionError
```

Comment:
Split this into 2 lines and append `# pylint: disable=redefined-builtin` to the new line. This should do it.

Reply:
used `requests.ConnectionError` directly